packages/store/src/lib/store.ts

```diff
@@ -12,6 +13,9 @@ export class Store<
   state!: State;
   private batchInProgress = false;
 
+  // TODO Infer the EntityAction type
+  private entityActions = new Subject<EntityAction<string | number>>();
```

Comment:
it should be actions in general. There is no coupling to entities.

```suggestion
  private actions = new Subject<Action<string | number>>();
```

Reply:
So the Action is carried in the `store` package ?

However the actions `add`, `set`, `remove` are related to the `entites `package.

```ts
export enum Actions {
  Set = 'Set',
  Add = 'Add',
  Update = 'Update',
  Remove = 'Remove',
}

export interface Action<IDType> {
  type: Actions;
  ids: IDType[];
}
```

Reply:
type is a general string.